nav-app/src/app/datatable/data-table.component.html

```diff
@@ -556,15 +556,15 @@
 
 <div class="legendBar" (click)="showingLegend = !showingLegend" *ngIf="!showingLegend && configService.getFeature('legend')">
     <img src="assets/icons/ic_keyboard_arrow_up_black_24px.svg" style="position: absolute; left:4px; top: 3px;">
-    <div class="noselect" style="padding:4px; font-size:14px; position: absolue; width: 100%; text-align:center">
+    <div class="noselect" style="padding:4px; font-size:14px; position: absolute; width: 100%; text-align:center">
         legend
     </div>
 </div>
 
 <div class="legend" *ngIf="showingLegend && configService.getFeature('legend')">
     <div class="legendBar" (click)="showingLegend = !showingLegend" style="position: absolute; top:0px; left:0px;">
         <img src="assets/icons/ic_keyboard_arrow_down_black_24px.svg" style="position: absolute; left:4px; top: 3px;">
-        <div class="noselect" style="padding:4px; font-size:14px; position: absolue; width: 100%; text-align:center">
+        <div class="noselect" style="padding:4px; font-size:14px; position: absolute; width: 100%; text-align:center">
```

Comment:
These changes _may_ be the cause of the legend issues mentioned previously:

![image](https://user-images.githubusercontent.com/28868888/124283467-ed7d2880-db19-11eb-8af0-40d223003297.png)